.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] One linter check is failing, the `stlylelint` check. All checks should be green inorder to achieve an error free code.Please fix this